backend/public-graph/graph/resolver.go

```diff
@@ -2494,8 +2514,9 @@ func (r *Resolver) ProcessPayload(ctx context.Context, sessionSecureID string, e
 	if sessionSecureID == "" {
 		return e.New("ProcessPayload called without secureID")
 	}
-	sessionObj := &model.Session{}
-	if err := r.DB.WithContext(ctx).Where(&model.Session{SecureID: sessionSecureID}).Limit(1).Take(&sessionObj).Error; err != nil {
+	var err error
+	var sessionObj *model.Session
+	if sessionObj, err = r.Store.GetSessionFromSecureID(ctx, sessionSecureID); err != nil {
```

Comment:
is it ok to use the cached version here? I would have thought there were some session attributes that could have changed if we're processing many payloads for the same session

Reply:
turned out is was not - though i didn't get to the bottom of what was important to not be cached (perhaps this would sometimes return `nil` after an initialize session?). reverted.